### PR TITLE
Course Evaluation API

### PR DESCRIPTION
**Pull request by kaalams**
### Description
Course Evaluation API and frontend. Frontend needs to be reworked to fit Semester.ly's current layout.

### CODE REVIEW

File: static/js/redux/init.jsx [hunk 1]
```diff
@@ -19,28 +19,42 @@ import { configureStore } from '@reduxjs/toolkit';
 import { Provider } from 'react-redux';
 import reducers from './reducers';
 import SemesterlyContainer from './ui/containers/semesterly_container';
-import { fetchMostClassmatesCount, handleAgreement, isRegistered } from './actions/user_actions';
+import { fetchMostClassmatesCount, handleAgreement, isRegistered } from './actions/user_actions
```
Comment (yesterd3v) [inline static/js/redux/init.jsx]: This is missing a closing quote, maybe my mistake when correcting conflicts

File: webpack.config.js [hunk 2]
```diff
@@ -65,79 +65,89 @@ const config = {
         // node_modules. That would take a long time.
         exclude: /node_modules/,
         // use the babel loader
-        loader: 'babel-loader',
+        loader: "babel-loader",
```
Comment (yesterd3v) [inline webpack.config.js]: I'm not sure why your autoformatter changed these all to double quotes. Can you see if you can fix this back to single quotes for all files?

File: semesterly/settings.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2017 Semester.ly Technologies, LLC
+ # Copyright (C) 2017 Semester.ly Technologies, LLC
```
Comment (yesterd3v) [inline semesterly/settings.py]: Can you remove this space as well?

**Review by yesterd3v — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 2 above)
Comment (kaalams) [reply webpack.config.js]: I honestly have no idea how

**Comment by JJamesWWang** [comment]
Let me know if this should be re-opened

**Pull request closed without merging**